src/test/java/edu/cornell/kfs/sys/web/mock/MockServiceEndpointBase.java

```diff
@@ -137,20 +134,20 @@ protected Optional<String> getNonBlankValueFromRequestUrlIfPresent(HttpRequest r
         }
     }
 
-    protected String getRequestContentAsString(HttpRequest request) {
+    protected String getRequestContentAsString(ClassicHttpRequest request) {
         return getRequestContent(request,
                 (contentStream) -> IOUtils.toString(contentStream, StandardCharsets.UTF_8));
     }
 
-    protected JsonNode getRequestContentAsJsonNodeTree(HttpRequest request) {
+    protected JsonNode getRequestContentAsJsonNodeTree(ClassicHttpRequest request) {
         return getRequestContent(request,
                 (contentStream) -> new ObjectMapper().readTree(contentStream));
     }
 
-    protected <R> R getRequestContent(HttpRequest request, IOExceptionProneFunction<InputStream, R> entityContentConverter) {
-        assertTrue("The request should have contained an entity", request instanceof HttpEntityEnclosingRequest);
+    protected <R> R getRequestContent(ClassicHttpRequest request, IOExceptionProneFunction<InputStream, R> entityContentConverter) {
+        assertTrue("The request should have contained an entity", request instanceof BasicClassicHttpRequest);
         
-        HttpEntityEnclosingRequest entityRequest = (HttpEntityEnclosingRequest) request;
+        BasicClassicHttpRequest entityRequest = (BasicClassicHttpRequest) request;
```

Comment:
Is this typecast needed anymore? It looks like ClassicHttpRequest already has an appropriate `getEntity()` method.

Reply:
ok